ansible/roles/test/files/ptftests/arista.py

```diff
@@ -175,6 +185,7 @@ def run(self):
                     'show ip route bgp' : bgp_route_v4_output,
                     'show ipv6 route bgp' : bgp_route_v6_output,
                 }
+            time.sleep(5)
```

Comment:
Why always sleep 5 seconds here?

Reply:
It's a polling which used to wait some status or protocal establisted between VMs and DUT.
An non-waiting polling will send lots of query in short time which is meaningless,
add a sleep 5 seconds to be an interval, help to prevent from resource wasting and garbage collecting.